libs/island-ui/core/src/lib/Footer/Footer.tsx

```diff
@@ -6,7 +6,8 @@ import { Logo } from '../Logo/Logo'
 import { Tiles } from '../Tiles/Tiles'
 import { Text } from '../Text/Text'
 import { Inline } from '../Inline/Inline'
-import { Icon } from '../Icon/Icon'
+import { Icon as IconDeprecated } from '../Icon/Icon'
```

Comment:
Why do we need to use the deprecated icon component?

Reply:
Because it contains the facebook icon, perhaps we could add that icon to the other Icon component then I can replace it

Reply:
I added the facebook icon from your PR and stopped using the deprecated Icon component in the Footer